main.go

```diff
@@ -53,6 +53,8 @@ func initInertia() *inertia.Inertia {
 			}
 			return url + entry, nil
 		})
+
+		i.ShareTemplateData("hmr", true)
```

Comment:
kinda feel dangerous for enabling this for production mode

Reply:
In production mode, there should be no `public/hot` file because the dev server is not running (and therefore this branch will never be reached), or am I missing something here?

Reply:
Ahh that makes sense. I missed that part